src/task.py

```diff
@@ -268,7 +361,26 @@ def get_ls_country_biome_kbas(kba_id):
                 get_ls_countries_biomes
             )
 
-        ls_countries_biomes_pas = landscapes.map(get_ls_countries_biomes_pas).flatten()
+        ls_countries_biomes_pas = (
+            landscapes.map(get_ls_countries_biomes_pas)
+            .flatten()
+            .filter(ee.Filter.gt("n", 0))
+            .select(
+                [
+                    "lsid",
+                    "lscountry",
+                    "ls_total_area",
+                    "lscountry_area",
+                    "areas",
```

Comment:
When I was looking at this with Dustin yesterday, the data structure was confusing (even for me, because I'm already starting to forget, and because much of this is new). Each polygon represents a country/landscape intersection; inside that is a nested object with state stats (called `areas`). Seems like it might be better to have each polygon represent a state/landscape intersection, and then it would be easier to aggregate by country, state, or landscape.

But I realize this would mean changing Eric's code and yours, and is probably not worth it for a short term goal. But maybe it's worth having a think about variable naming, and/or documentation (e.g. are the state areas for the whole state, or just the part covered by the current landscape polygon, and if the latter does that mean that some stats may not add up from landscapes to state level, etc.).